### PR TITLE
[test] Add Likwid test to the health suite

### PR DESCRIPTION
**Pull request by ChristopherBignamini**

### CODE REVIEW

File: cscs-checks/microbenchmarks/cpu/likwid/likwid.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ def __init__(self):
 
         self.executable = 'likwid-bench'
 
-        self.num_tasks = 0
+        self.num_tasks = 4
```
Comment (victorusu) [inline cscs-checks/microbenchmarks/cpu/likwid/likwid.py:18]: @ChristopherBignamini, is there any specific reason why you are selecting the number 4?

Comment (ChristopherBignamini) [reply cscs-checks/microbenchmarks/cpu/likwid/likwid.py]: As discussed, I will set it to 1.

File: cscs-checks/microbenchmarks/cpu/likwid/likwid.py [hunk 2]
```diff
@@ -42,11 +43,11 @@ def __init__(self):
...
         }
 
-        self.maintainers = ['SK']
-        self.tags = {'benchmark', 'diagnostic'}
+        self.maintainers = ['SK', 'CB']
+        self.tags = {'benchmark', 'diagnostic', 'health'}
 
-        bw_pattern = sn.extractsingle(r'MByte/s:\s*(?P<bw>\S+)',
-                                      self.stdout, 'bw',  float)
+        bw_pattern = sn.min(sn.extractall(r'MByte/s:\s*(?P<bw>\S+)',
```
Comment (victorusu) [inline cscs-checks/microbenchmarks/cpu/likwid/likwid.py:49]: This is a great solution! Get the minimum value!

**Review by victorusu — APPROVED** [review]
`lgtm`

**Review by vkarak — APPROVED** [review]

**Comment by vkarak** [comment]
@ChristopherBignamini Is this PR only about Likwid or generally about CPU sanity checks?

**Comment by ChristopherBignamini** [comment]
> @ChristopherBignamini Is this PR only about Likwid or generally about CPU sanity checks?

The PR is about CPU sanity checks in general I would say but all the identified tests so far are already part of the likwid check list.

**Pull request merged**